.vscode/settings.json

```diff
@@ -27,5 +27,8 @@
     80,
     90
   ],
-  "editor.tabSize": 2
+  "editor.tabSize": 2,
+  "files.associations": {
+    "*.primitives": "jsonc"
```

Comment:
Should this just be `json`? I don't believe Aeson supports `jsonc` (although that would be nice).

Reply:
@martijnbastiaan yes, it should. This is a result of habit. I'll change

Also :rofl: 
> As for me, I like big diffs and I cannot lie.